CITATION.cff

```diff
@@ -0,0 +1,53 @@
+cff-version: 1.2.0
+title: DLA-Future
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Alberto
+    family-names: Invernizzi
```

Comment:
Are affiliations not included by choice or they are not supported?

Reply:
See https://github.com/eth-cscs/DLA-Future/pull/1082#issuecomment-1895837863.

> It's also possible to add affiliation and email for each author. I left them out initially since they have a tendency to become out of date (and orcid is more likely to be up to date), but will happily include them as well for whoever wants it there.

So, supported, initially left out, but happy to add them. I have no objection either way.

Reply:
Sorry I missed your comment. You are right it is very difficult to track the affiliation of people that is not contributing anymore.